src/routes/ComparisonsRouter.ts

```diff
@@ -1,20 +1,29 @@
 import express from 'express';
 import pg from '../database/knex';
 import { Comparison } from '@src/models/Comparisons';
+import ComparisonsService from '@src/services/ComparisonsService';
 
 
 const comparisonRouter = express.Router();
 
 // Define routes
 comparisonRouter.get('/', async function (req, res) {
-  const comparisons: Comparison[] = await pg('comparisons').select('*');
+  const comparisons: Comparison[] = await ComparisonsService.getAllComparisons();
```

Comment:
I imagine the UI flow is that the client presents one random comparison at a time. So they would probably call this endpoint. It sounds like a lot to return all comparisons in one API call.

Reply:
Agreed, we probably don't need this endpoint to get all comparisons.

For the UI flow, I think what I'm imagining the following:
1. User selects a category
2. Based on the category, the backend fetches a list of subjects (get Subjects by category Id)
3. We can either return a list of tuple subjects back to the UI, or all of the subjects.
4. UI prompts users to select a subject. 
5. User votes and then we return the comparison object.